src/Components/Patient/DailyRoundListDetails.tsx

```diff
@@ -58,13 +64,13 @@ export const DailyRoundListDetails = (props: any) => {
         setIsLoading(false);
       }
     },
-    [consultationId, dispatch, id]
+    [consultationId, id]
   );
   useAbortableEffect(
     (status: statusType) => {
       fetchpatient(status);
     },
-    [dispatch, fetchpatient]
+    [fetchpatient]
```

Comment:
`useAbortableEffect` can be removed when using useQuery

Reply:
Lately i was also bit confused how useQuery was managing the states and what things are to be removed, but after seeing this, now i am getting idea what things are to be removed.